code/game/jobs/department.dm

```diff
@@ -44,7 +44,7 @@
 	In future, we will implement largescale missions and research contracts to earn money, and then set it
 	to a much lower starting value
 	*/
-	account_initial_balance = 2000000
+	account_initial_balance = 300000
```

Comment:
```suggestion
	account_initial_balance = 600000
```
...how about we still give captains some leeway?

Reply:
To what? There's 15 hours of payments to fully staffed dependants within 300k, and the likelihood of techno/IH both being fully staffed for 15 hours is literally impossible.

Captain using the money for frivolous shit is still doable. 600k is enough to both break the trade beacons, AND pay fully staffed dependants for 20 hours.

300k means breaking the trade beacons isn't as brute force, and more careful purchases while still being able to fund dependants for 6-10 hours.

Reply:
Just so the captain still has some *leeway*. They are a rich person. If they can't flex they're free 1.7 millions, let them flex atleast 300k. If they misuse it we'll have to talk to them instead.

Money is one of the more distinct captain's powers.

Reply:
According to Mycah's calculations Captain needs around 120k for 6h round. That means Captain is free to flex their 180k.
I don't feel like 480k flex-money is right.

Reply:
yeah 480k spare credits is waaaaaaaaaaaaay too much leeway, you can still absolutely wreck the game's economy with that and brute force both guild and moebius trees

Reply:
fair